src/main/java/hudson/plugins/analysis/graph/GraphConfiguration.java

```diff
@@ -433,6 +493,8 @@ private void reset() {
         dayCount = DEFAULT_DAY_COUNT;
         graphType = DEFAULT_GRAPH;
         useBuildDate = DEFAULT_USE_BUILD_DATE;
+        parameterName = null;
```

Comment:
I said that as soon as you serialize PARAMETER_VALUE  and PARAMETER_NAME to file of cookie the difference disappears.
I do not see any benefits of changing this code but feel free to change it as you see fit.